packages/template-retail-react-app/app/pages/product-list/index.jsx

```diff
@@ -414,6 +429,15 @@ const ProductList = (props) => {
             ) : (
                 <>
                     <AbovePageHeader />
+                    {/* Banner */}
+                    {instagramPromoBanner && !pageError && (
```

Comment:
We need to also check there's no error because some cases when the shopper context changes, data is returned but the API will have a 500 error

Reply:
Are these changes in this page removed before merging?

Reply:
I was thinking these changes would stay in the page as it would demonstrate in the retail react app how customers can change the UI based on shopper context